docs/source/user-guide/latest/compatibility.md

```diff
@@ -20,70 +20,105 @@ under the License.
 # Compatibility Guide
 
 Comet aims to provide consistent results with the version of Apache Spark that is being used.
+This guide documents known differences and limitations.
 
-This guide offers information about areas of functionality where there are known differences.
+---
 
 ## Parquet
 
 Comet has the following limitations when reading Parquet files:
 
-- Comet does not support reading decimals encoded in binary format.
-- No support for default values that are nested types (e.g., maps, arrays, structs). Literal default values are supported.
+- Decimal values encoded using binary format are not supported.
+- Default values for nested types (e.g., maps, arrays, structs) are not supported.
+  Literal default values are supported.
```

Comment:
Not sure if  we are covering Decimal -> Binary format documentation in this change ?